build/requirements_lock_3_12.txt

```diff
@@ -492,6 +492,10 @@ nvidia-nvshmem-cu12==3.2.5 ; sys_platform == "linux" \
     # via
     #   -r build/requirements.in
     #   jax-cuda12-plugin
+nvidia-nvshmem-cu13==0.0.0a0 \
```

Comment:
This will obviously be removed if/when we regenerate these, which is not infrequently.